client/src/components/_common/Button/Button.jsx

```diff
@@ -57,28 +84,36 @@ const Button = ({
 
   return (
     <button
-      className={`c-button ${buttonTypeClass} ${buttonSizeClass}`}
+      className={`${buttonRootClass} ${buttonTypeClass} ${buttonSizeClass}`}
```

Comment:
**Intent:** To have any class names, static or dynamic, be in one location (see lines above).